Applications/Games/Anno 2070/Uplay/script.js

```diff
@@ -0,0 +1,26 @@
+include(["engines", "wine", "quick_script", "uplay_script"]);
+include(["engines", "wine", "verbs", "corefonts"]);
+include(["engines", "wine", "plugins", "virtual_desktop"]);
+
+var installerImplementation = {
+    run: function () {
+        new UplayScript()
+            .name("Anno 2070")
+            .editor("Ubisoft")
+            .applicationHomepage("http://anno-game.ubi.com/anno-2070/en-US/")
+            .author("Zemogiter")
+            .wineVersion("3.20")
```

Comment:
Can we use always the latest development version?

Reply:
Let me remark this again: we should really try to use the latest stable/devel/staging as much as possible to avoid fragmenting the installed Wine versions too much.